workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/notification-templates/template-8/en-US.html

```diff
@@ -0,0 +1,11 @@
+<p>
+	Hi [%BUSINESSEVENT_AUTHOR_FIRST_NAME%],
+</p>
+
+<p>
+	The target date for your business event, [%BUSINESSEVENT_NAME%], has passed. Please update the status of your project. Your open support tickets are prioritized based on the status of your Business Event. Record the <a href="[$PORTAL_URL$]/project/#/[%ACCOUNTENTRYTOBUSINESSEVENTS_ACCOUNTENTRY_EXTERNALREFERENCECODE%]/business-events/[%BUSINESSEVENT_ID%]?openModal=goLiveEvent">actual go live date</a> and share any feedback you have or <a href="[$PORTAL_URL$]/project/#/[%ACCOUNTENTRYTOBUSINESSEVENTS_ACCOUNTENTRY_EXTERNALREFERENCECODE%]/business-events/[%BUSINESSEVENT_ID%]/edit">update your event</a> with a new date to let us know how Liferay can continue to support you. Thank you for your attention to this matter.
```

Comment:
business event is capitalized in once instance and non-capitalized in another. Probably should always be lowercase?